db/functions/redundant.py

```diff
@@ -40,6 +59,48 @@ class GreaterOrEqual(DBFunction):
         hints.mathesar_filter,
     ])
 
-    @staticmethod
-    def to_sa_expression(value1, value2):
-        return value1 >= value2
+    def unpack(self):
+        param0 = self.parameters[0]
+        param1 = self.parameters[1]
+        return Or([
+            Greater([param0, param1]),
+            Equal([param0, param1]),
+        ])
+
+
+class StartsWithCaseInsensitive(RedundantDBFunction):
+    id = 'starts_with_case_insensitive'
+    name = 'starts with (case insensitive)'
```

Comment:
If this name is meant to be used in the frontend, please remove ` (case insensitive)` from the `name` parameter.

Reply:
It is. If the suffix is removed, it will have the same display name as a regular (case-sensitive) `starts with` filter: a user won't be able to distinguish the two.

Reply:
There shouldn't be a case-sensitive "starts with" in the frontend, only the case insensitive version. 

We don't want to overwhelm users with a lot of similar filtering options, it will make finding the filter they want harder. And non-technical users may not know what "case sensitive" refers to.